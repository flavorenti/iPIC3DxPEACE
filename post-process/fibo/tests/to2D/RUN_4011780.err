slurmstepd-irene8002: error: task/cgroup: task[0] unable to set taskset '0x0'
+ SCRIPT_PID=27219
+ set +x
+ /bin/bash -x /tmp/jobstart.27174
+ unset _mlshdbg
+ '[' 1 = 1 ']'
+ case "$-" in
+ set +x
+ unset _mlshdbg
+ ml python
+ module ml python
+ unset _mlshdbg
+ '[' 1 = 1 ']'
+ case "$-" in
+ set +x
load module python/2.7.14 (Python)
+ unset _mlshdbg
+ return 0
+ set -x
+ ccc_mprun -E--exclusive -n 1 python -u test_to2D.py /ccc/scratch/cont005/gen12428/lavorenf/Mercury_SaeInit/PR1/Normal-newbox/run0/data
+ ccc_mprun -E--exclusive -n 1 python -u test_to2D.py /ccc/scratch/cont005/gen12428/lavorenf/Mercury_SaeInit/PR1/Normal-newbox/run1/data
+ wait
+ ccc_mprun -E--exclusive -n 1 python -u test_to2D.py /ccc/scratch/cont005/gen12428/lavorenf/Mercury_SaeInit/PR1/Normal-newbox/run2/data
+ ccc_mprun -E--exclusive -n 1 python -u test_to2D.py /ccc/scratch/cont005/gen12428/lavorenf/Mercury_SaeInit/PR1/Normal-newbox/run3/data
slurmstepd-irene8002: error: Detected 1 oom-kill event(s) in step 4011780.2 cgroup. Some of your processes may have been killed by the cgroup out-of-memory handler.
srun: error: irene8002: task 0: Out Of Memory
slurmstepd-irene8002: error: Detected 1 oom-kill event(s) in step 4011780.0 cgroup. Some of your processes may have been killed by the cgroup out-of-memory handler.
srun: error: irene8002: task 0: Out Of Memory
slurmstepd-irene8002: error: Detected 1 oom-kill event(s) in step 4011780.1 cgroup. Some of your processes may have been killed by the cgroup out-of-memory handler.
srun: error: irene8002: task 0: Out Of Memory
slurmstepd-irene8002: error: Detected 1 oom-kill event(s) in step 4011780.3 cgroup. Some of your processes may have been killed by the cgroup out-of-memory handler.
srun: error: irene8002: task 0: Out Of Memory
+ exit 0
