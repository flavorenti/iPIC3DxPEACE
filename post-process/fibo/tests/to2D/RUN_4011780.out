('iPIC3D> cell number               :  ', (960, 768, 768))
('iPIC3D> cell number               :  ', (960, 768, 768))
('iPIC3D> cell number               :  ', (960, 768, 768))
('iPIC3D> cell number               :  ', (960, 768, 768))
('iPIC3D> domain size               :  ', (82.5, 66.0, 66.0))
('iPIC3D> mesh spacing              :  ', (0.0859375, 0.0859375, 0.0859375))
('iPIC3D> periodicity               :  ', (False, False, False))
('iPIC3D> time step                 :  ', 0.5)
('iPIC3D> species                   :  ', ['e0', 'i1'])
('iPIC3D> domain size               :  ', (82.5, 66.0, 66.0))
('          e0 charge-over-mass           :  ', -100.0)
('iPIC3D> mesh spacing              :  ', (0.0859375, 0.0859375, 0.0859375))
('          i1 charge-over-mass           :  ', 1.0)
('iPIC3D> periodicity               :  ', (False, False, False))
('LOAD->', 1000)
('iPIC3D> time step                 :  ', 0.5)
('iPIC3D> species                   :  ', ['e0', 'i1'])
('          e0 charge-over-mass           :  ', -100.0)
('          i1 charge-over-mass           :  ', 1.0)
('LOAD->', 5000)
('iPIC3D> domain size               :  ', (82.5, 66.0, 66.0))
('iPIC3D> mesh spacing              :  ', (0.0859375, 0.0859375, 0.0859375))
('iPIC3D> periodicity               :  ', (False, False, False))
('iPIC3D> time step                 :  ', 0.5)
('iPIC3D> species                   :  ', ['e0', 'i1'])
('          e0 charge-over-mass           :  ', -100.0)
('          i1 charge-over-mass           :  ', 1.0)
('LOAD->', 3000)
('iPIC3D> domain size               :  ', (82.5, 66.0, 66.0))
('iPIC3D> mesh spacing              :  ', (0.0859375, 0.0859375, 0.0859375))
('iPIC3D> periodicity               :  ', (False, False, False))
('iPIC3D> time step                 :  ', 0.5)
('iPIC3D> species                   :  ', ['e0', 'i1'])
('          e0 charge-over-mass           :  ', -100.0)
('          i1 charge-over-mass           :  ', 1.0)
('LOAD->', 7000)
('CUT->', 7000)
('PRINT->', 7000)
('DELETE->', 7000)
COMP.TIME [seg7000] = 641.746 s

('LOAD->', 7100)
('CUT->', 5000)
('PRINT->', 5000)
('DELETE->', 5000)
COMP.TIME [seg5000] = 650.309 s

('LOAD->', 5100)
('CUT->', 1000)
('PRINT->', 1000)
('CUT->', 3000)
('PRINT->', 3000)
('DELETE->', 1000)
COMP.TIME [seg1000] = 659.160 s

('LOAD->', 1100)
('DELETE->', 3000)
COMP.TIME [seg3000] = 661.351 s

('LOAD->', 3100)
('CUT->', 7100)
('PRINT->', 7100)
('DELETE->', 7100)
COMP.TIME [seg7100] = 640.537 s

('LOAD->', 7200)
('CUT->', 1100)
('PRINT->', 1100)
('DELETE->', 1100)
COMP.TIME [seg1100] = 644.426 s

('LOAD->', 1200)
('CUT->', 5100)
('PRINT->', 5100)
('DELETE->', 5100)
COMP.TIME [seg5100] = 685.112 s

('LOAD->', 5200)
('CUT->', 3100)
('PRINT->', 3100)
('DELETE->', 3100)
COMP.TIME [seg3100] = 719.522 s

('LOAD->', 3200)
('CUT->', 7200)
('PRINT->', 7200)
('DELETE->', 7200)
COMP.TIME [seg7200] = 673.281 s

('LOAD->', 7300)
('CUT->', 1200)
('PRINT->', 1200)
('DELETE->', 1200)
COMP.TIME [seg1200] = 687.827 s

('LOAD->', 1300)
('CUT->', 5200)
('PRINT->', 5200)
('DELETE->', 5200)
COMP.TIME [seg5200] = 662.065 s

('LOAD->', 5300)
('CUT->', 3200)
('PRINT->', 3200)
('DELETE->', 3200)
COMP.TIME [seg3200] = 679.740 s

('LOAD->', 3300)
('CUT->', 7300)
('PRINT->', 7300)
('DELETE->', 7300)
COMP.TIME [seg7300] = 629.090 s

('LOAD->', 7400)
('CUT->', 1300)
('PRINT->', 1300)
('DELETE->', 1300)
COMP.TIME [seg1300] = 634.388 s

('LOAD->', 1400)
('CUT->', 5300)
('PRINT->', 5300)
('DELETE->', 5300)
COMP.TIME [seg5300] = 648.002 s

('LOAD->', 5400)
('CUT->', 3300)
('PRINT->', 3300)
('DELETE->', 3300)
COMP.TIME [seg3300] = 652.310 s

('LOAD->', 3400)
('CUT->', 7400)
('PRINT->', 7400)
('DELETE->', 7400)
COMP.TIME [seg7400] = 646.661 s

('LOAD->', 7500)
('CUT->', 1400)
('PRINT->', 1400)
('DELETE->', 1400)
COMP.TIME [seg1400] = 625.138 s

('LOAD->', 1500)
('CUT->', 5400)
('PRINT->', 5400)
('DELETE->', 5400)
COMP.TIME [seg5400] = 611.237 s

('LOAD->', 5500)
('CUT->', 3400)
('PRINT->', 3400)
('DELETE->', 3400)
COMP.TIME [seg3400] = 655.042 s

('LOAD->', 3500)
('CUT->', 1500)
('PRINT->', 1500)
('DELETE->', 1500)
COMP.TIME [seg1500] = 629.495 s

('LOAD->', 1600)
('CUT->', 7500)
('PRINT->', 7500)
('DELETE->', 7500)
COMP.TIME [seg7500] = 666.875 s

('LOAD->', 7600)
('CUT->', 5500)
('PRINT->', 5500)
('DELETE->', 5500)
COMP.TIME [seg5500] = 673.076 s

('LOAD->', 5600)
('CUT->', 3500)
('PRINT->', 3500)
('DELETE->', 3500)
COMP.TIME [seg3500] = 655.941 s

('LOAD->', 3600)
('CUT->', 1600)
('PRINT->', 1600)
('DELETE->', 1600)
COMP.TIME [seg1600] = 628.258 s

('LOAD->', 1700)
('CUT->', 7600)
('PRINT->', 7600)
('DELETE->', 7600)
COMP.TIME [seg7600] = 629.355 s

('LOAD->', 7700)
('CUT->', 5600)
('PRINT->', 5600)
('DELETE->', 5600)
COMP.TIME [seg5600] = 633.601 s

('LOAD->', 5700)
('CUT->', 3600)
('PRINT->', 3600)
('DELETE->', 3600)
COMP.TIME [seg3600] = 674.539 s

('LOAD->', 3700)
('CUT->', 1700)
('PRINT->', 1700)
('DELETE->', 1700)
COMP.TIME [seg1700] = 669.476 s

('LOAD->', 1800)
('CUT->', 5700)
('PRINT->', 5700)
('DELETE->', 5700)
COMP.TIME [seg5700] = 676.803 s

('LOAD->', 5800)
('CUT->', 7700)
('PRINT->', 7700)
('DELETE->', 7700)
COMP.TIME [seg7700] = 792.168 s

('LOAD->', 7800)
('CUT->', 3700)
('PRINT->', 3700)
('DELETE->', 3700)
COMP.TIME [seg3700] = 746.885 s

('LOAD->', 3800)
('CUT->', 1800)
('PRINT->', 1800)
('DELETE->', 1800)
COMP.TIME [seg1800] = 792.033 s

('LOAD->', 1900)
('CUT->', 5800)
('PRINT->', 5800)
('DELETE->', 5800)
COMP.TIME [seg5800] = 752.922 s

('LOAD->', 5900)
('CUT->', 7800)
('PRINT->', 7800)
('DELETE->', 7800)
COMP.TIME [seg7800] = 841.581 s

('LOAD->', 7900)
('CUT->', 3800)
('PRINT->', 3800)
('DELETE->', 3800)
COMP.TIME [seg3800] = 722.055 s

('LOAD->', 3900)
########## ########## ########## ########## ########## ########## ########## ##########
Execution Sum Up
########## ########## ########## ########## ########## ########## ########## ##########
Jobid     : 4011780
Jobname   : 2D
User      : lavorenf
Account   : gen12428@xlarge
Limits    : time = 2:47:00 , memory/task = 26800 Mo
Date      : submit = 02/02/2022 16:47:51 , start = 02/02/2022 16:47:52 , end = 02/02/2022 19:34:52
Execution : partition = xlarge , QoS = normal@xlarge , Comment = (null)
Resources : ntasks = 4 , cpus/task = 20 , ncpus = 80 , nodes = 1
   Nodes=irene8002 CPU_IDs=8-87 Mem=2144000 GRES=
 
Memory / step 
--------------
                          Resident Size (Mo)                     Virtual Size (Go)
JobID            Max     (Node:Task)       AveTask    Max  (Node:Task)            AveTask
-----------      ------------------------  -------    --------------------------  -------

Accounting / step 
------------------

JobID           JobName                  Ntasks  Ncpus Nnodes  Layout       Elapsed               User         System     Eff  State
------------    ------------             ------  ----- ------  -------      -------               -------      ------     ---  -----
4011780         2D                            -     80      1        -     01:56:33   (100%)            -            -      -  -
4011780.batch   batch                         1     80      1  Unknown     01:56:33   (100%)     00:00:00     00:00:00      0  RUNNING
4011780.extern  extern                        1     80      1  Unknown     01:56:33 (100.0%)     00:00:00     00:00:00      0  RUNNING
4011780.0       python                        1     20      1   BBlock     01:54:36  (98.3%)     00:34:28     00:22:31    2.4  OUT_OF_MEMORY
4011780.1       python                        1     20      1   BBlock     01:56:19  (99.7%)     00:34:46     00:22:00    2.4  OUT_OF_MEMORY
4011780.2       python                        1     20      1   BBlock     01:52:44  (96.7%)     00:35:57     00:19:55    2.4  OUT_OF_MEMORY
4011780.3       python                        1     20      1   BBlock     01:56:27  (99.9%)     00:34:45     00:21:15    2.4  OUT_OF_MEMORY
########## ########## ########## ########## ########## ########## ########## ##########
